src/v/coproc/gen.json

```diff
@@ -1,30 +0,0 @@
-{
```

Comment:
Since you are up to date with dev, you are deleting the current version of gen.json which you shouldn't be doing. This is the proper RPC IDL file you need to work with to complete this change

Reply:
Yes, you're right, that PR was created before your PR was created and merged, for that reason I handle that file in that way, I need to update it